include/alpaka/event/EventUniformCudaHipRt.hpp

```diff
@@ -123,6 +123,12 @@ namespace alpaka
     };
     namespace traits
     {
+        //! The CUDA/HIP RT device event device type trait specialization.
+        template <>
```

Comment:
clang-format says:
```suggestion
        template<>
```

Reply:
what version of clang do you use ?
`clang-format` from clang 11 complains about unknown options in the format file :-(

Reply:
We use clang-format 12 (specifically 12.0.1), as documented [here](https://github.com/alpaka-group/alpaka/blob/develop/CONTRIBUTING.md).

Reply:
Fixed, thanks.